internal/worker/filenotifywatcher/watcher.go

```diff
@@ -95,20 +96,31 @@ type Watcher struct {
 	logger Logger
 }
 
+// NewWatcher returns a new FileWatcher that watches the given fileName in the
+// given path.
 func NewWatcher(fileName string, opts ...Option) (FileWatcher, error) {
 	o := newOption()
 	for _, opt := range opts {
 		opt(o)
 	}
 
+	// Ensure that we create the watch path.
```

Comment:
Can't see which commit explains this addition.

Reply:
This was creating noise in the agent test suite, which made it even more than infuriating to diagnose what the underlying problem was.